doc/enterprise-shield/o_enterprise_shield_checklist.md

```diff
@@ -0,0 +1,55 @@
+
+
+About Enterprise Shield™
+-----------------------------------------------
+
+Enterprise Shield™ protects your trusted network by initiating the connection from the internal trusted network towards the DMZ. Until now, allowing access to a trusted network while still maintaining security behind the firewall has been impossible because of the necessity to open ports to accept incoming connections. For most administrators, opening a port to the outside world is a necessary but undesirable solution because it instantly increases vulnerability to outside hacks and attacks. Companies are reluctant to open up ports in firewalls because each open port is another potential attack vector for malicious users. Using Enterprise Shield™, you can close all of your inbound ports while still allowing clients to initiate connections.
+
+Traditionally, web architectures require that you open inbound ports to allow connectivity to internal systems and services in your trusted network. As with any web architecture, a typical KAAZING Gateway configuration (without Enterprise Shield™) must open ports to allow TCP, HTTP, or WebSocket connectivity through a firewall, as shown in the following figure:
+
+![Gateway Topology with Ports Open to the Trusted Network](../images/f-dmz-trusted-top.png)
+
+**Figure 1: A KAAZING Gateway Topology with Ports Open to the Trusted Network**
+
+Using Enterprise Shield™, you can close all inbound ports in your firewall, thus closing the entry points available to untrusted users and eliminating your attack surface vulnerability.
+
+To implement Enterprise Shield™, you configure a Gateway in the DMZ, which receives a reverse connection from within the trusted network. With this architecture, a client can talk to the back-end service or message broker through a firewall. Another benefit of Enterprise Shield™ is that your architecture remains valid, without requiring changes. For example, neither the client nor the back-end service or message broker are aware of the reverse connection because it is completely transparent to the rest of the architecture. Clients that are outside the firewall connect as usual to the DMZ Gateway.
+
+![Simple Topology Showing a Reverse Connection](../images/f-dmz-trustednetwork-860-02.png)
+
+**Figure 2: Closed Inbound Ports in an Enterprise Shield™ Topology**
+
+Adding Enterprise Shield™ to your architecture requires only a few simple modifications. Instead of a single Gateway in the DMZ, you add another Gateway to your trusted network. Then, you only need to make a few changes to the two Gateway configurations to reverse the connection. Other parts of the architecture, such as the client and the back-end service or message broker, observe no apparent differences between a configuration with Enterprise Shield™ or one without, making the reverse connection completely transparent to the endpoints of the configuration.
```

Comment:
Yes!  That's been in the doc that way for awhile. How did we keep missing that? Fixed.

 "Instead of a single Gateway in your trusted network, you add another Gateway in the DMZ. "